resources/core/charts/monitoring/charts/alertmanager/README.md

```diff
@@ -20,68 +20,105 @@ metadata:
     release: {{ .Release.Name }}
   name: alertmanager-{{ .Release.Name }}
 data:
-  alertmanager.yaml: {{ toYaml .Values.config | b64enc | quote }}
+  alertmanager.yaml: |-
+    {{ include "alertmanager.yaml.tpl" . | b64enc }}
 {{- range $key, $val := .Values.templateFiles }}
   {{ $key }}: {{ $val | b64enc | quote }}
 {{- end }}
 ```
 
-The next section explains the Alertmanager configuration.
+Secret `data:` is an encoded ```alertmanager.yaml``` file which contains all the configuration for alerting notifications.
+
+The next section explains how to configure Alertmanager for enabling alerting notifications.

```

Comment:
Please move this sentence to the next section and change it this way:
``This section explains how to configure Alertmanager to enable alerting notifications.``